tests/charm++/jacobi3d/jacobi3d.C

```diff
@@ -217,6 +232,9 @@ class Jacobi: public CBase_Jacobi {
     double *temperature;
     double *new_temperature;
 
+    int msg_overhead;
+    char* temp;
```

Comment:
I think these names could be more descriptive, maybe something like `msg_overhead_size` and `msg_overhead_data` or similar.